### PR TITLE
Fix upload limit

### PR DESCRIPTION
**Pull request by flozia**
Sets the upload limit actually to `10 MB` and adds instructions on how to test `/upload`.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ use actix_web::{
 };
 use log::info;
 
-const MAX_PAYLOAD_SIZE: usize = 80000000; // 10 Megabyte
+const MAX_PAYLOAD_SIZE: usize = 10000000; // 10 Megabyte
```
Comment (bakulf) [inline src/main.rs:10]: ok.. this is hard. Should this be 10485760? nobody cares too much.

**Review by bakulf — APPROVED** [review]

File: src/main.rs (hunk 1 above)
Comment (jbuck) [reply src/main.rs:10]: Could this be a configurable parameter, with a default of 10MB? We usually configure things like this through environment variables

**Review by jbuck — APPROVED** [review]
lgtm, but I'd like a way to configure the maximum upload size too

File: src/main.rs (hunk 1 above)
Comment (flozia) [reply src/main.rs:10]: Yes! Adding support for env variables with PR #9.

**Pull request merged**